src/util.ts

```diff
@@ -88,3 +89,19 @@ export function createOutputRetainingCompilerHost(
     outputFiles.set(fileName, content);
   }
 }
+
+/**
+ * True if we are running on Windows (32 or 64 bit).
+ */
+export const isWindows: boolean = os.platform() === 'win32';
+
+/**
+ * Returns the input with line endings normalized to '\n', when running on Windows.
+ * Returns the string as is, for any other OS.
+ */
+export function normalizeLineEndings(input: string): string {
+  if (isWindows)
```

Comment:
Not really, it's unnecessary on all other platforms, so I added it to save processing time. But I can remove it if you say it's not relevant.